pyInterface/genPseudoData.py

```diff
@@ -167,7 +167,7 @@ def norm(c):
 				                                prodKinNames,
 				                                decayKinNames,
 # TODO: FILL THESE
-				                                {},
+				                                {"mass":( 1000. * massRange[0], 1000. * massRange[1] )},
```

Comment:
Please format it like this: `{ "mass": (1000. * massRange[0], 1000. * massRange[1]) }`.